File: Python Programming/problems on string manipulation/problem104.py
'''
104. Write a program which accept string from user and display only digits from that string.
	Input :   “marve89llous121” 
	Output :   89121 
	Input :   “Demo” 
	
	Author : Milind Nalawade
	
	Date : 15/07/2021
		
	language : python 3.9.0

'''

#====
#Logic
#====
def Digit(str):
	for i in str:
		if i.isdigit():
			print(i,end=" ")
#=============
#Entry point function
#=============
def main():
	str=input("\n Enter your string : ")
	Digit(str);
#=====
#starter
#=====
if __name__=="__main__":
	main()

File: C Programming/problems on strings maipulation/problem104.c
/*
104. Write a program which accept string from user and display only digits from that string.
	Input :   “marve89llous121” 
	Output :   89121 
	Input :   “Demo” 
	
	Author : Milind Nalawade
	
	Date : 15/07/2021
		
	language : C programming

*/


#include<stdio.h>
#include<string.h>

void Digit(char* arr)
{
	
	for (int i=0;i<strlen(arr);i++)
	{
		if(arr[i]>=49&&arr[i]<=57)
		{
			printf(" %c",arr[i]);
		}
	}	
}

//Entry point function

int main()
{	
	char arr[20];
	int ret=0;
	printf("\n Enter your string : ");
	scanf("%[^'\n']s",arr);
	Digit(arr);
	return 0;
}	